controllers/operands/kubevirt.go

```diff
@@ -280,9 +280,11 @@ func hcWorkloadUpdateStrategyToKv(hcObject *hcov1beta1.HyperConvergedWorkloadUpd
 			*kvObject.BatchEvictionSize = *hcObject.BatchEvictionSize
 		}
 
-		if size := len(hcObject.WorkloadUpdateMethods); size > 0 {
+		filterWorkloadUpdateMethods := filterWorkloadupdatemethods(hcObject.WorkloadUpdateMethods)
```

Comment:
Please rename to prevent name collision with the function, and improve readability.

Reply:
done